collections/testtscripts.js

```diff
@@ -54,8 +55,7 @@ Meteor.methods({
         completedCounts[elem] = 1;
       }
       else {
-        completedCounts[elem] += 1;
-      }
+        completedCounts[elem] += 1; }
```

Comment:
Can you change the formatting back to how it was, with the `}` on the next line?